docs/diagrams/class/Exceptions.puml

```diff
@@ -0,0 +1,51 @@
+@startuml
```

Comment:
Good job on he UML diagram for the DG!

Reply:
Thanks for adding the UML diagram for the DG!